### ISSUE TITLE
Community Exchange Introduction/Tracking:

### ISSUE THREAD

**Issue by ericdrosado** [Portfolio-Shop/portfolioshop#84]
👋 Hi @sudip-mondal-2002 ,

I am your GitHub mentor for the GitHub Education, [Community Exchange](https://airtable.com/shrRJ3AvEZQohOdGV) (CX) project. I'll be collaborating with you on preparing your repo for CX. 

You mentioned in your submission that you wanted to submit a `Learn + Collaborate` repo. This means that you would like to teach students, step by step, how to build this project as well as invite other students to collaborate and add features to this repo. I will generate issues, which will provide guidance on how to prepare your repo for a `Learn + Collaborate` CX submission on June 1, 2022.

This issue will serve as a tracking issue to track all issues related to CX. I recommend creating a new branch for every issue and opening a pull request to track changes so we can effectively collaborate with each other and merge changes when you and I feel like those changes are ready to be merged on your primary branch. 

If you have any questions or concerns, please feel free to leave a comment on this issue or any of the other issues that are generated. 

I look forward to working with you :octocat:

**Comment by sudip-mondal-2002** [comment]
I have fixed all these issues, please look into them and let me know if I need to do any further tasks.

**Comment by ericdrosado** [comment]
Congratulations @sudip-mondal-2002 for completing all of the issues! 🎉 

I will contact you next week on next steps in this issue.

**Comment by sudip-mondal-2002** [comment]
Hi @ericdrosado any updates on this? Thank you!

**Comment by sudip-mondal-2002** [comment]
Hi @ericdrosado I can't add organization repositories in the community exchange. Was it intended or is it a bug?
![image](https://user-images.githubusercontent.com/74463091/171786880-58808179-bb21-463c-8547-838f5e678e84.png)

**Comment by ericdrosado** [comment]
@sudip-mondal-2002 good catch! We did not account for Organization level submissions. 

Let me check with the engineers to see if we can adjust this. I'll post updates in this issue.

**Comment by sudip-mondal-2002** [comment]
Any updates on this @ericdrosado sir?

**Comment by ericdrosado** [comment]
@sudip-mondal-2002 it looks like we will be unable to accept Org level repositories at this time. The plan will be to consider this for a future iteration, but we won't be able to add this feature by launch. 

If you would like to submit your repo before launch, I would recommend [transferring the repo to your name](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#transferring-a-repository-owned-by-your-organization).

If this is something you are not interested in doing, we completely understand. This won't prevent you from receiving GitHub Swag for participating.

**Comment by sudip-mondal-2002** [comment]
Yes, @ericdrosado. It's not possible for me to transfer the repo to my account. We'll do it in the next iteration. sure. what about the survey form? Should I fill it out now?
![image](https://user-images.githubusercontent.com/74463091/172206949-9045b6ba-adb9-4881-8d08-bb3778339628.png)
